app/react/Markdown/markdownDatasets.js

```diff
@@ -116,7 +116,8 @@ export default {
 
   getMetadataValue(state, { property, dataset = 'default' }) {
     const data = state.page.datasets.get(dataset);
-    const mos = data ? data.getIn(['metadata', property]).toJS() : [];
+    const propertyExists = data ? data.hasIn(['metadata', property]) : false;
```

Comment:
The ternary operation is not necesary here, it could be a simple condition like 
`const propertyExists = data && data.hasIn(['metadata', property]);`, please could you replace it?

Reply:
Thank you Mercy! Line changed.